### PR TITLE
Updated NGSI-LD Use-cases to adapt to recent version

### PR DESCRIPTION
**Pull request by Neeraj-Nec**
Correct NGSILD usecase

### CODE REVIEW

**Comment by showersky** [comment]
It looks good. If you have already tested those NGSI-LD applications, I can merge this PR and validate it from my side.

**Comment by Aniket-NEC** [comment]
@showersky I have tested the use-cases and they work fine. 

> I can merge this PR and validate it from my side.

Yes, it would be great if you could merge this pull request. I will soon raise a PR for updated tutorial for all the use-cases. 
Thank you.

File: common/ngsi/ngsiclient.go [hunk 1]
```diff
@@ -936,7 +936,7 @@ func (nc *NGSI10Client) SubscribeLdContext(sub *LDSubscriptionRequest, requireRe
         }
 	req.Header.Add("Content-Type", "application/json")
 	req.Header.Add("Accept", "application/ld+json")
-	req.Header.Add("Link", "<https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld>; rel=\"http://www.w3.org/ns/json-ld#context\"; type=\"application/ld+json\"")
+	req.Header.Add("Link", "<https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld>; rel=\"http://www.w3.org/ns/json-ld#context\"; type=\"application/ld+json\"")
```
Comment (naveennec) [inline common/ngsi/ngsiclient.go:939]: please check it as currently we don't support 1.3 and correct the extension.

File: doc/en/source/ngsildbasediotintegration.rst [hunk 2]
```diff
@@ -351,7 +352,7 @@ How to Fetch data from Orion-LD to FogFlow
 		  -H 'Accept: application/ld+json' \
 		  -H 'fiware-service: openiot' \
 		  -H 'fiware-servicepath: /' \
-		  -H 'Link: <https://fiware.github.io/data-models/context.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
+		  -H 'Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
```
Comment (naveennec) [inline doc/en/source/ngsildbasediotintegration.rst:355]: Check everywhere 1.3 is not more supported by FogFlow , it should not be there.

**Review by naveennec — CHANGES_REQUESTED** [review]

**Pull request merged**